tests/TestCase/Mailer/EmailTest.php

```diff
@@ -2712,6 +2712,7 @@ public function testJsonSerialize()
         ];
         $this->assertEquals($expected, $result);
 
+        debug(unserialize(serialize($this->CakeEmail)));
         $result = json_decode(json_encode(unserialize(serialize($this->CakeEmail))), true);
```

Comment:
I think we'll have to add same serialization / unserialization feature to ViewBuilder too.